openedx/core/djangoapps/edx_global_analytics/utils/utilities.py

```diff
@@ -79,46 +79,17 @@ def get_previous_month_start_and_end_dates():
 
 def get_coordinates_by_ip():
     """
-    Gather coordinates by server IP address with FreeGeoIP service.
+    Gather coordinates by server IP address with ip-api service.
+    This endpoint is limited to 150 requests per minute from an IP address.
     """
-    try:
-        ip_data = requests.get('https://freegeoip.net/json')
-        latitude, longitude = ip_data.json()['latitude'], ip_data.json()['longitude']
-        return latitude, longitude
+    latitude, longitude = '', ''
 
-    except requests.RequestException as error:
-        logger.exception(error.message)
-        return '', ''
+    ip_data = requests.get('http://ip-api.com/json')
 
+    if ip_data.status_code == 200:
```

Comment:
should `ip_data.json()` be also checked before we start taking values by the keys?